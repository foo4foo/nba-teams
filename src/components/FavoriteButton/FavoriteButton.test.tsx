import React from 'react'
import renderer from 'react-test-renderer'
import { FavoriteButton } from './'

describe('FavoriteButton', () => {
  it('renders successfuly', () => {
    const tree = renderer
      .create(<FavoriteButton isFavorite onClick={() => console.log('test')} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
