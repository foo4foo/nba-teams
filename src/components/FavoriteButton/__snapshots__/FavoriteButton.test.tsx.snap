// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoriteButton renders successfuly 1`] = `
<svg
  aria-label="Favorite"
  className="StyledIcon-ofa7kd-0 cQhyoo"
  onClick={[Function]}
  style={
    Object {
      "cursor": "pointer",
    }
  }
  viewBox="0 0 24 24"
>
  <path
    d="M1,8.4 C1,4 4.5,3 6.5,3 C9,3 11,5 12,6.5 C13,5 15,3 17.5,3 C19.5,3 23,4 23,8.4 C23,15 12,21 12,21 C12,21 1,15 1,8.4 Z"
    fill="none"
    stroke="#000"
    strokeWidth="2"
  />
</svg>
`;
